nginx-proxy/entrypoint.sh

```diff
@@ -15,8 +15,12 @@ else
   export TEMPLATE="/etc/nginx/default.conf.template"
 fi
 
+for error in 400 404 413 422 500 502 503; do
+  curl -fsSL "https://${STATIC_PAGES_BUCKET}.s3.amazonaws.com/errors/${error}.html" -o "/app/public/system/errors/${error}.html"
```

Comment:
As far as I can tell the error pages for nginx are copied from an S3 bucket, instead of by invoking the `generate_error_pages` rake task on the FL app. Could we not copy the error pages in the FL app docker container the same way?

Reply:
That would mean avoiding having them on the nginx container.

Reply:
I think this would be problematic. The NGINX container boots before the app, so there would be a period of time where it doesn't have access to error pages since we're waiting for the app to boot and mount the volume.

NGINX does require access to these error pages, since it is intercepting a certain set of errors.